src/lammpsio/snapshot.py

```diff
@@ -437,6 +439,33 @@ def has_mass(self):
         """
         return self._mass is not None
 
+    @property
+    def bonds(self):
+        """:class:Bonds: bonds."""
+        if not self.has_bonds():
+            pass
+        return self._bonds
+
+    @bonds.setter
+    def bonds(self, value):
+        if value is not None:
+            if not isinstance(value, Bonds):
+                raise ValueError("bonds must be :class:Bonds:")
```

Comment:
rst doesn't work in exceptions
```suggestion
                raise ValueError("bonds must be Bonds")
```